README.md

```diff
@@ -66,9 +67,24 @@ The collection is also available as `ansible-collection-theforeman-foreman` from
 
 After installing the RPM, you can use the modules in the same way as when they are installed directly from Ansible Galaxy.
 
+## Using the modules
+
+Please see [the Plugin Index](https://theforeman.org/plugins/foreman-ansible-modules/) for the list of plugins in the theforeman.foreman collection.
+You'll find examples on the individual plugin pages.
+
+## Switching from katello-cvmanager
```

Comment:
I'd prefer not to have this so prominently here. But have  migration page in the docs might be a good idea.

Reply:
Agreed, a separate page would be nicer. Once #958 is done I can whip something up since I will permanently switch from it to the collection then.
Until we have one maybe keep the section? While it was never supported, a lot of Satellite admins I met used it (because it was so much more pleasant than them having to write shells scripts with hammer commands).